### DISCUSSION TITLE
甲骨文云

### DISCUSSION THREAD

**Discussion by wumianye8** [mack-a/v2ray-agent#667] [Q&A]
我的甲骨文换了几个不同的域名，包括.eu.org 和.ml等，均出现“TLS安装失败，请检查acme日志”的提示，之后就结束安装，请问这是怎么回事？只有甲骨文出现这个情况，别的vps服务器一切正常无报错进度

**Comment by mack-a** [comment]
有没有具体的报错信息

**Reply by wumianye8** [reply]
已解决！看了“八合一脚本使用注意事项”后才知道是甲骨文还有一个额外的防火墙要手动关闭。谢谢老大，万分感谢！！！
